<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnDangXuat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Đăng Xuất</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTenNhanVien">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>511</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>MENU CHỨC NĂNG </string>
    </property>
    <widget class="QPushButton" name="btnQuanLyNhanVien">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quản lý nhân viên</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQuanLyXe">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quản lý xe</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDangKiVe">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Đăng ký vé</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGiaHanVe">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Gia hạn vé</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnQuanLyVe">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quản lý giá vé</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBaoCaoDoanhThu">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Báo cáo doanh thu</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnQuanLyVaoRa">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quản lý vào/ra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MÀN HÌNH CÔNG VIỆC</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
